sys/log/full/src/log.c

```diff
@@ -648,15 +690,13 @@ log_append_mbuf_typed_no_free(struct log *log, uint8_t module, uint8_t level,
      * maximum entry length was checked.
      */
     len = os_mbuf_len(om);
```

Comment:
I realize you did not add this code, but this line is wrong.  It should use `OS_MBUF_PKTLEN()` instead of `os_mbuf_len()`.  I.e., we care about the length of the entire chain, not just the first buffer.

I don't think this needs to be fixed in this PR, but we should create an issue or a new PR soon.

Reply:
Not applicable here.